app/views/foods/shopping_list.html.erb

```diff
@@ -0,0 +1,13 @@
+<!-- app/views/foods/shopping_list.html.erb -->
+<div class="container">
+  <h1 class="head-section">Shopping List</h1>
+  <ul class="shopping_list">
+    <% @missing_foods.each do |food| %>
+      <li><%= food.name %></li>
+    <% end %>
+  </ul>
+  <div class="total">
+    <p>Total Food Items: <%= @total_items %></p>
+    <p>Total Price: <%= number_to_currency(@total_price) %></p>
+  </div>
+</div>
```

Comment:
[OPTIONAL] Please make sure to fix this issue as it is preventing your app from functioning as expected, this is mentioned as OPTIONAL only because it was missed by the previous reviewer
the generated shopping list for a given recipe doesn't display any foods while the recipe has a list of food.
![Screenshot from 2024-02-10 23-28-48](https://github.com/faisalakb/food-recipe-group/assets/59707859/b4b2e9da-b0f7-4924-99d2-7c97778d7d2b)
![Screenshot from 2024-02-10 23-28-38](https://github.com/faisalakb/food-recipe-group/assets/59707859/44b8b755-e2d3-43ef-9adf-3bdade359f91)